core/controller/src/main/scala/whisk/core/controller/Controller.scala

```diff
@@ -181,11 +181,13 @@ object Controller {
         val instance = if (args.length > 0) args(1).toInt else 0
 
         // initialize the runtimes manifest
-        if (config.isValid && ExecManifest.initialize(config)) {
+        val execManifest = ExecManifest.initialize(config)
+        if (config.isValid && execManifest.isSuccess) {
             val port = config.servicePort.toInt
             BasicHttpService.startService(actorSystem, "controller", "0.0.0.0", port, new ServiceBuilder(config, instance, logger))
         } else {
-            logger.error(this, "Bad configuration, cannot start.")
+            logger.error(this, "Bad configuration, cannot start." +
+              (if (execManifest.isFailure) " Invalid runtimes.manifest: " + execManifest.failed.get.toString else ""))
```

Comment:
Yes, I could make it a separate log message to appear before the line saying `Bad configuration, cannot start.` . The other failures in config with required properties appear before too.